### ISSUE TITLE
Minor grammar blooper in "Deferred Reference"

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6272]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)| @hynek reported|
|-|-|
|Trac ID|trac#6272|
|Type|enhancement|
|Created|2013-01-21 16:56:31Z|
|Branch|https://github.com/twisted/twisted/tree/defer-typo-6272|

"If the errback does returns a Failure or raise an exception,"

There’s a "does" too much and an "s" missing.

Attachments:

* [typo-in-defer.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/b81/b815d3d0d85f528649c3d0cd2156437d13f80b28/a1f4d9597d43f9afe663aef963c848eb69d61362.patch) (824 bytes) - added by hynek on 2013-01-21 16:56:48Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__6272 6272
type__enhancement enhancement
reporter__hynek hynek
priority__normal normal
milestone__ 
branch__branches_defer_typo_6272 branches/defer-typo-6272
branch_author__hynek__tom_prince hynek, tom.prince
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1358787391000000 1358787391000000
changetime__1358828616000000 1358828616000000
version__None None
owner__tom_prince tom.prince

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_1"></a>|@tomprince commented|
|-|-|

(In [36873]) Branching to defer-type-6272.

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_2"></a>|@tomprince set owner to @tomprince<br>@tomprince set status to `assigned`|
|-|-|

I think the fix in the patch is backwards. The does is intentional.

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)|@tomprince removed owner<br>@tomprince set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

> (In [36873]) Branching to defer-type-6272.

Note this is not actually the branch name :/

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set owner to @tomprince<br>@exarkun set status to `new`|
|-|-|

> I think the fix in the patch is backwards. The does is intentional.

I agree.  I think the fix in the branch to make the "does return" agreement is correct.  Please merge (with a misc fragment).

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_6"></a>|@tomprince set status to `closed`|
|-|-|

(In [36877]) Merge branches/defer-typo-6272: Fix minor typo in "Deferred Reference"

Author: hynek, tom.prince
Reviewers: tom.prince, exarkun
Fixes: [#6272](https://github.com/twisted/twisted/issues/6272)
